src/instanceMethods/keydown-handler.js

```diff
@@ -74,10 +65,6 @@ const keydownHandler = (instance, e, dismissWith) => {
   // ARROWS - switch focus between buttons
   } else if ([...arrowKeysNextButton, ...arrowKeysPreviousButton].includes(e.key)) {
     handleArrows(e.key)
-
-  // ESC
-  } else if (escKeys.includes(e.key)) {
```

Comment:
This block should not be removed. I asked about removing the `escKeys` variable, nothing else. Please revert this logic with:

```js
} else if (e.key === 'Escape') {